pkgs/development/python-modules/dataclasses-json/default.nix

```diff
@@ -21,10 +24,19 @@ buildPythonPackage rec {
     marshmallow-enum
   ];
 
+  checkInputs = with python3Packages; [
```

Comment:
The tests would be enabled with https://github.com/NixOS/nixpkgs/pull/115114/commits/4053c81e2991c933b775c524ebb8f2dd9717d41e

Reply:
Oh, thanks! I'll make this a draft until that gets merged in and I'll rebase off of your changes.